### PR TITLE
Add pacman package support

### LINKED ISSUE #814: Add support for arch
# Feature Request
How difficult would it be to add support for arch? `fpm` should be able to make pacman packages right?

### PR DESCRIPTION
**Pull request by uberhacker**
Fixes #814.

Thank you so much for contributing code to lando!

We will get to your PR as soon as we can but in the meantime you might want to
check to make sure you have done the following. **The more boxes you can check
the easier it will be for us to merge in your changes with confidence**

- [x] My code passes relevant CI status checks
- [ ] My code includes a test if applicable ([see here](https://docs.devwithlando.io/dev/testing.html))
- [x] My code includes an update to the `CHANGELOG` ([see here](https://github.com/kalabox/lando/tree/master/docs/changelog))
- [x] My code includes documentation updates if relevant.
- [ ] I have pinged [a project committer](https://docs.devwithlando.io/contrib/contributing.html#committers) when I think my code is ready for prime time.

If you are unclear about any of the above please add comments below so we can
improve our process.

### CODE REVIEW

**Comment by pirog** [comment]
@uberhacker def a great start here. Here are a list of some other things we should do before we merge this in

- [ ] Update the branch with the latest from `master` (might need to do this a few times over the lifecycle of this PR)
- [ ] Add an entry to the [changelog](https://github.com/lando/lando/blob/master/docs/changelog/2018.md) with the issue number/link/description like the other items listed
- [ ] Update our [`.travis.yml`](https://github.com/lando/lando/blob/master/.travis.yml#L9) with an additional `pacman` flavored build 
- [ ] Update the [install docs](https://github.com/lando/lando/blob/master/.travis.yml#L9) with `pacman` related things. Particularly obvious places are the linux install steps and a development release link
- [ ] Get a bunch of people to test this out and see if it works

**Comment by pirog** [comment]
@uberhacker definitely getting further down the road. Looks like travis doesnt like the pacman build. 
https://travis-ci.org/lando/lando/jobs/359691265#L3363

possibly because it doesnt have bsdtar available? Would be a great entry to add to our https://github.com/lando/lando/blob/master/scripts/install-deps.sh#L46 script for arch

File: scripts/install-deps.sh [hunk 1]
```diff
@@ -44,15 +44,24 @@ echo "Mmmmm this ${FLAVOR} flavor is so delcious"
...
       curl \
       file \
-      rpm
+      rpm \
+      ruby
+      gem install rdoc fpm
     ;;
   fedora)
-    echo "Not implemented yet!"
+  redhat)
```
Comment (inverse) [inline scripts/install-deps.sh]: Broken for me
`
stderr: ./scripts/install-deps.sh: line 62: syntax error near unexpected token `)'
./scripts/install-deps.sh: line 62: `  redhat)'
`

Comment (inverse) [reply scripts/install-deps.sh]: Further more should fedora support be in another PR?

Comment (uberhacker) [reply scripts/install-deps.sh]: @inverse: I appreciate the feedback.  Yeah, this should be separated out, although redhat and fedora should have identical sections (in theory, at least).

File: docs/changelog/2018.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ v3.0.0-beta.38 - In Development
 * Improved `linux` Docker start command handling [#659](https://github.com/lando/lando/issues/659)
 * Improved handling and documentation of upload issues on Windows [#396](https://github.com/lando/lando/issues/396)
 * Improved and simplified our examples [#798](https://github.com/lando/lando/issues/798) [#801](https://github.com/lando/lando/issues/801)
+* Add Arch/Manjaro pacman package build support [#814](https://github.com/lando/lando/issues/814)
```
[the remark this answers is not in the document]
Comment (uberhacker) [reply docs/changelog/2018.md]: What if we add Arch based distro support?

**Comment by pirog** [comment]
@uberhacker i made the changes below (based on your last commit) and was able to build the whole thing on fresh 17.10 box (similar-ish to travis) with just `yarn pkg:build`

```diff
diff --git a/scripts/install-deps.sh b/scripts/install-deps.sh
index 07836c4e..2e64a449 100755
--- a/scripts/install-deps.sh
+++ b/scripts/install-deps.sh
@@ -46,7 +46,6 @@ echo "Mmmmm this ${FLAVOR} flavor is so delcious"
 case $FLAVOR in
   arch)
     pacman -Syu ruby
-    gem install rdoc fpm
     ;;
   debian)
     apt-get update && apt-get -y --force-yes install \
@@ -55,14 +54,12 @@ case $FLAVOR in
       curl \
       file \
       rpm \
-      ruby
-      gem install rdoc fpm
+      ruby \
+      ruby-dev
     ;;
   fedora)
-    ;&
+    ;;
   redhat)
-    yum install ruby
-    gem install rdoc fpm
     ;;
   osx)
     echo "Not implemented yet!"
```

**Comment by pirog** [comment]
@uberhacker guessing you might have accidentally added in your self-update stuff to this branch

**Comment by uberhacker** [comment]
@pirog: Thank you for the feedback.  I have pushed the changes you suggested.  Still can't get by these daggum tests, however.

**Comment by uberhacker** [comment]
Closing due to boneheaded mistake.

**Pull request closed without merging**